sdk/python/foundation-models/system/inference/image-classification/image-classification-batch-endpoint.ipynb

```diff
@@ -239,7 +239,7 @@
     "\n",
     "We can provide input images to batch inference either in a folder containing images or in a csv file containing \"image\" named column having images in base64 format.\n",
     "\n",
-    "Note: If job failed with error Assertion Error (`The actual length exceeded max length 100 MB`) then please try with less number of input images or use ImageFolder Input mode."
+    "The deployment in the \"Create batch deployment\" section below takes the argument `mini_batch_size`, which is the number of CSV files processed by the model in a single mini_batch."
```

Comment:
is my understanding correct ?
mini_batch_size= number of rows in single csv file ?
number of mini batches == number of csv files ?

I am little confused here as different csv can have different number of rows.